source/build_wf_pages.py

```
@@ -0,0 +1,89 @@
+            ]
+        wf_data["badges"] = "\n".join(badges)
+        wf_data["description"] = repo["description"]
+        wf_data["topics"] = repo["topics"]
+        wf_data["linting"] = (
+            "{bdg-success}`passed`"
+            if repo["linting"] == None
+            else "{bdg-danger}`failed`"
+        )
+        wf_data["formatting"] = (
+            "{bdg-success}`passed`"
+            if repo["formatting"] == None
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)